app/views/submissions/edit.html.erb

```diff
@@ -3,66 +3,54 @@
+        <th>Criteria</th>
+        <th>Description</th>
+        <th>Score</th>
       </tr>
     </thead>
-
     <%= form_for @submission, :method=>:patch do |f|%>
-      <tbody>
+    <tbody>
+      <% @assignment.rubric_fields.each do |field|%>
+        <% grading_field = @submission.grading_fields.where(rubric_field_id: field.id).first%>
+        <% if grading_field.nil? %>
+          <% grading_field = @submission.grading_fields.new(rubric_field_id: field.id) %>
+        <% end %>
         <tr>
-          <% @assignment.rubric_fields.each do |field| %>
-            <% grading_field = @submission.grading_fields.where(rubric_field_id: field.id).first%>
-            <% if grading_field.nil? %>
-              <% grading_field = @submission.grading_fields.new(rubric_field_id: field.id) %>
-            <% end %>
-            <%= f.fields_for :grading_fields, grading_field do |builder|%>
-              <td>
-                <div class='form-group'>
-                  <%= builder.number_field :score, :min=>0, :max=>10, :class=>"form-control input-sm number_field", :placeholder=>"score"%>
-                </div>
-                <div class='form-group'>
-                  <%= builder.text_area :comment, :class=>"form-control input-sm", :placeholder=>"comment"%>
-                </div>
-                <%= builder.hidden_field :rubric_field_id%>
-              </td>
-            <% end %>
-          <% end %>
-          <td>
-            <div class='form-group'>
-              <%= f.number_field :self_assessment_grade, :class=>"form-control input-sm self_grade"%>
-            </div>
-          </td>
+          <td><%= field.name %></td>
+          <td></td>
+          <%= f.fields_for :grading_fields, grading_field do |builder|%>
           <td>
             <div class='form-group'>
-              <%= f.number_field :ta_grade, :class=>"form-control input-sm total_grade"%>
-            </div>
-          </td>
-          <td>
-            <div class='form-group'>
-              <%= f.number_field :final_grade, :class=>"form-control input-sm final_grade"%>
-            </div>
-          </td>
-
-          <td>
-            <div class='form-group'>
-              <%= f.number_field :sa_points, :class=>"form-control input-sm sa_points"%>
+              <%= builder.number_field :score, :min=>0, :max=>10, :class=>"form-control input-sm number_field", :placeholder=>"score"%>
             </div>
           </td>
+          <%= builder.hidden_field :rubric_field_id%>
+          <% end %>
         </tr>
-
-        <tr>
+      <% end %>
+      <tr>
+        <td>Self Assessment Grade</td>
+        <td></td>
+        <td><%= f.number_field :self_assessment_grade, :class=>"form-control input-sm self_grade"%></td>
+      </tr>
+      <tr>
+        <td>TA Grade</td>
+        <td></td>
+        <td><%= f.number_field :ta_grade, :class=>"form-control input-sm total_grade"%></td>
+      </tr>
+      <tr>
+        <td>Self Assessment points</td>
```

Comment:
The original one had two: one for "self assessment grade" and other for "self assessment points". I wasn't sure what the difference was, but thought I need to keep it since it was there in the first place.